defaults/main.yml

```diff
@@ -19,6 +19,9 @@ gitlab_runner_executable: "{{ gitlab_runner_package_name }}"
 # Maximum number of global jobs to run concurrently
 gitlab_runner_concurrent: "{{ ansible_processor_vcpus }}"
 
+# Defines the interval length, in seconds, between the runner checking for new jobs.
```

Comment:
Defines the interval length, in seconds, between the runner checking for new jobs. The default value is 3. If set to 0 or lower, the default value is used.

Reply:
Happy to change this, but I found using the GitLab documentation verbatim here confusing for the user due to the repeated use of `default value`:

There is a default value for *THIS* Ansible role (which is `0`, meaning use the GitLab runner default); and a default value for GitLab runner (which your comment describes)

Reply:
Updated with slightly tweaked wording, hopefully clearer now.